tools/release/release_utils.sh

```diff
@@ -70,6 +70,22 @@ function publishImages {
     docker logout
 }
 
+function publishPackages{
+    local curTag=$1
+    local baseTag=$2
+    local ghcrUser=$3
+    local ghcrPassword=$4
+    local images=$5
+    docker login ghcr.io -u $ghcrUser -p $ghcrPassword
+    for image in $images
+    do
+        echo "push image: $image"
+        docker tag $image:$baseTag "ghcr.io/"$image:$curTag
+        retry 5 docker push "ghcr.io/"$image:$1curTag   
```

Comment:
1. Remove meaningless spaces at the end of this line.
2. It should be `$curTag` not `$1curTag`